bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -660,3 +761,9 @@ async def cog_unload(self) -> None:
             self.autoreview_loop.cancel()
 
         self.prune_talentpool.cancel()
+
+        self.bot.tree.remove_command(
+            self.nominate_user_context_menu.name,
+            guild=discord.Object(Guild.id),
+            type=self.nominate_user_context_menu.type
+        )
```

Comment:
Since we're removing it on unload, does that mean we should add it back on load somehow?

Reply:
We only remove the handler, surely it gets re-added when the setup function is called again?

This does not synchronise through to Discord AFAIK, it just stops erroneous dispatches for commands to cogs that are not loaded.

Reply:
Ah right, simple thing to check using the extensions command I guess, would be interested in how it works